tests/test_logger.py

```
@@ -0,0 +1,36 @@
+        ("core", "INFO", "¡Hola!", "core: INFO - ¡Hola!"),
+        ("система", "ОШИБКА", "не удалось", "система: ОШИБКА - не удалось"),
+        # Very long strings
+        (
+            "n" * 1000,
+            "L" * 1000,
+            "m" * 1000,
+            f'{"n"*1000}: {"L"*1000} - {"m"*1000}',
+        ),
+    ],
+)
+def test_log_formats_message(
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)